clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -972,16 +975,28 @@ private Record nextFetchedRecord() {
         private List<ConsumerRecord<K, V>> fetchRecords(int maxRecords) {
             if (isFetched)
                 return Collections.emptyList();
+            if (nextInlineException != null) {
```

Comment:
@lindong28 You asked for a review. I reviewed and raised a concern. Instead of taking 5 minutes to address it, you spent weeks contesting it and trying to escalate to others. As of now, the concern is unaddressed, so why should I merge?